docs/control-structures.rst

```diff
@@ -51,24 +54,20 @@ write::
     }
 
 The names of output parameters can be omitted.
-The output values can also be specified using ``return`` statements.
-The ``return`` statements are also capable of returning multiple
-values, see :ref:`multi-return`.
-Return parameters are initialized to zero; if they are not explicitly
+The output values can also be specified using ``return`` statements,
+which are also capable of :ref:`returning multiple values<multi-return>`.
```

Comment:
`returning multiple values`?

Reply:
Can you elaborate on that? :)

Reply:
Ah, when I wrote that I missed the word `values`